installer/conf/omsagent.d/mongo.conf

```diff
@@ -19,13 +19,20 @@
   type tail
   path /var/log/mongodb/mongod.log
   tag  oms.api.MongoDBlog
-  format /(?<timestamp>[^ ]*) (?<severity>[A-Z]) (?<component>[^ ]*)  \[(?<context>[^\]]*)\] (?<message>.*)/
+  format /(?<timestamp>[^ ]*) (?<severity>[A-Z]) (?<component>(-\s*|([^ ]*)))  \[(?<context>[^\]]*)\] (?<message>.*)/
+</source>
+
+#for logs with component: COMMAND
+<source>
```

Comment:
Is this re-tailing the file? Is there a way to filter for only messages that contain Command? Additionally can we change the variable to querytime_seconds and convert ms to seconds?